docs/source/_static/css/custom.css

```diff
@@ -180,3 +180,37 @@ table.rows tr:nth-child(odd) {
     background-color: #FFFFFF;
     text-align: right;
 }
+
+
+/* reduce size of dropdown boxes and change color */
+
+/* Title text attributes */
+details.dropdown .summary-title {
+  font-weight: bold !important;
+  font-family: var(--pst-font-family-monospace) !important;
+}
+
+/* Space of box around the text */
+details.dropdown summary {
+  padding: 0.4rem !important;
+}
+
+/* Vertical spacing of arrow */
+details.dropdown .summary-up, details.dropdown .summary-down {
+  top: 0.5em !important;
+}
+
+/* Vertical space between dropdown boxes */
+.mb-3, .my-3 {
+  margin-bottom: 0.2rem !important;
+}
```

Comment:
At the moment this change does not have an effect when building the documentation via ``make html``. For some reason, it is read when building via ``tox -e sphinx``. Apparently, the ``mb-3`` class comes from [bootstrap](https://getbootstrap.com/docs/4.0/utilities/spacing/) where m stands for margin, b for bottom and 3 for 1rem.